lib/package_source.py

```diff
@@ -10,6 +10,26 @@
 from lib import utils
 
 
+def set_dest_dir(source, directory, local_copy_subdir_name):
```

Comment:
Could you document this method and mention the git repository is created from a local directory if it does not exist?